src/tests/api/wellKnown.test.ts

```diff
@@ -0,0 +1,14 @@
+import { assert200OK, assert201Created, assert400BadRequest, assert401Unauthorized } from "../../utils/apiCommonAssertions";
+import { authorizationGet, claimedIdentityPost, sessionPost, stubStartPost, tokenPost, userInfoPost, wellKnownGet } from "../../utils/apiTestSteps";
+
+describe("Happy Path Get WellKnown", () => {
+
+	it("Check wellKnown", async () => {
+		const response = await wellKnownGet();
+         
+        
+		assert200OK({ status: response.status, statusText: response.statusText });
```

Comment:
Should we do some validation that we're exposing the key types we'll need to expose to work correctly?